ci/conda_requirements.txt

```diff
@@ -14,3 +14,4 @@ ipywidgets
 numpydoc
 sphinx
 sphinx_bootstrap_theme
+pandoc
```

Comment:
pandoc is required for the nbsphinx. since nbsphinx is not in conda, we pip install it, but better to have pandoc from conda?
or do you think there's a better way to do it?

Reply:
this is good.